### DISCUSSION TITLE
ACC indicator version 1.33

### DISCUSSION THREAD

**Discussion by Kutabu** [ochadenas/cpudefense#134] [General]
The indicator for the ACC chip was changed together with the MEM upgrade. It is good that the thick frame no longer suggests an interaction (which does not exist as of version 1.33).

The block at the bottom is a very good indicator. Nicely done! But the fact that the number of blocks increases with the upgrade is very irritating. The upgrade level of the chip is already indicated by + , v and &.
The multiple blocks suggest that at some point more than one of the blocks will have to fill up. Even though the flash-over effect looks very cool when saving the first attacker, I would stick to a single block. It's less irritating and here I think "less is more".

**Comment by ochadenas** [comment]
Ah. Sorry, that's a bug. In fact, the indicator is handled by the same logic as that for MEM, but of course there should only be one slot.

Moving this discussion to the "issues" page.
